tests/install_upgrade_operators/product_upgrade/conftest.py

```
@@ -169,7 +170,9 @@ def updated_cnv_subscription_source(cnv_subscription_scope_session, cnv_registry
 
 
 @pytest.fixture()
```

Ruff (0.15.10)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)